### ISSUE TITLE
Support for alts

### ISSUE THREAD

**Issue by joelarmstrong** [ComparativeGenomicsToolkit/coordination#10]
Cactus should have native support for alts (maybe enabled by default?) in some way, which would treat alts as separate genomes. They can probably still be treated the same way in the HAL output, though.

The same code might be able to be used for diploid assemblies--it's exactly the same principle.

**Comment by ifiddes** [comment]
Would this work for N-Ploidy? (Many alts for one locus).

Also, I think this would be good for any alignment that uses hg38.

Ian Fiddes

**Comment by joelarmstrong** [comment]
Yeah, it should handle more-than-diploid as well, I think some of the hg38 loci have more than 1 alt.

I'm not really sure how this should work for "fragmented" diploid assemblies though. Where you might have enough information to phase part of the chromosome, so you end up with 2 copies, but broken up into tons of contigs which may not necessarily start/end in the same place in both copies. Would that be best as tens of thousands of separate "genomes"? Kinda weird.
